src/js/components/agencyV2/visualizations/ObligationsByAwardType.jsx

```diff
@@ -59,6 +61,30 @@ export default function ObligationsByAwardType({
     const outerStrokeWidth = 3;
     const innerRadius = outerRadius - (outerStrokeWidth * 2);
 
+    const mapToFullCategoryName = (categoryType) => `All ${categoryType.charAt(0).toUpperCase()}  ${categoryType.slice(1)}`;
```

Comment:
Wondering if some of these formatting functions could be moved to a helpers file (and given unit tests)

Reply:
Good point.  I moved to helpers file and created unit tests.